doc/manual/src/release-notes/rl-next.md

```diff
@@ -1 +1,4 @@
 # Release X.Y (202?-??-??)
+
+- URLs in Nix now support [percent-encoded](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) paths. The command line will percent-
+  encode non-URI characters automatically.
```

Comment:
```suggestion
- [Flake references](@docroot@/command-ref/new-cli/nix3-flake.md#flake-references) now support [percent-encoded](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) paths. The command line will percent-encode non-URI characters automatically.
```

Reply:
Included in a79affd6d (and fixed the bit about the command-line which is wrong)